packages/react-router/modules/resolveLocation.js

```diff
@@ -0,0 +1,38 @@
+import resolvePathname from 'resolve-pathname'
+
+const resolveLocation = (location, base) => {
+  base = !base ? '/' : base
```

Comment:
Let's use a new constant instead of re-assigning the value of one of the function arguments.